src/main/java/com/google/opensesame/objectify/ObjectifyWebFilter.java

```diff
@@ -0,0 +1,7 @@
+package com.google.opensesame.objectify;
+
+import com.googlecode.objectify.ObjectifyFilter;
+import javax.servlet.annotation.WebFilter;
+
+@WebFilter(urlPatterns = {"/*"})
```

Comment:
Maybe we can look into what Servlets need to be  Filtered? Not a big deal tbh, this works fine.

Reply:
I'm thinking we should keep this as-is. Otherwise it'll require constant upkeep as new servlets are introduced